addons/udes_stock/models/stock_move.py

```diff
@@ -185,9 +193,8 @@ def split_out_incomplete_move(self, **kwargs):
         total_done_qty = sum(mls.mapped("qty_done"))
```

Comment:
Maybe this should be total quantity of the mls instead of only qty done when mls_to_keep is not None.